web/src/upload.js

```diff
@@ -86,6 +92,12 @@ export function get_item(key, config, file_id) {
                 return $(
```

Comment:
I think here also `upload_banner_cancel_button` name would be less confusing.

Reply:
Changed it, @brijsiyag could you please review again, thanks :)